pkg/kubelet/hyper/hyperclient.go

```diff
@@ -809,15 +809,34 @@ func (client *HyperClient) ListServices(podId string) ([]HyperService, error) {
 }
 
 func (client *HyperClient) UpdateServices(podId string, services []HyperService) error {
```

Comment:
As issue 136 said, you should never use any types from `pkg/kubelet/hyper/types.go`, you should always return structs defined in `pkg/kubelet/hyper/types/types.pb.go` instead.

And you should also check if the returned struct in  `pkg/kubelet/hyper/types/types.pb.go` is the same in `pkg/kubelet/hyper/types/types.pb.go`, if not, you need to change the code in `pkg/kubelet/hyper/hyper.go` to make it work with structs in `types.pb.go`